packages/gatsby-plugin-sharp/src/plugin-options.js

```diff
@@ -2,7 +2,8 @@ const _ = require(`lodash`)
 
 /// Plugin options are loaded onPreBootstrap in gatsby-node
 const pluginDefaults = {
-  forceBase64Format: false,
+  base64Width: 20,
+  forceBase64Format: ``, // valid formats: png,jpg,webp
```

Comment:
This got omitted somehow, I probably didn't click on start-review...
Can't we change this to undefined instead? Technically this becomes an enum with 3 values then. We could add validation later on.
```suggestion
```

Reply:
@wardpeet You want to change the value of empty string to `undefined`? I've just matched the value to what is already used for `toFormat`

https://github.com/gatsbyjs/gatsby/blob/1419afe21a48fc4d75909e054760693fadc8675f/packages/gatsby-plugin-sharp/README.md#L142-L143

https://github.com/gatsbyjs/gatsby/blob/1419afe21a48fc4d75909e054760693fadc8675f/packages/gatsby-plugin-sharp/src/plugin-options.js#L26-L27

`toFormatBase64` like `toFormat` is the per image option, while `forceBase64Format` is a global default setting (perhaps the name doesn't communicate that well? It's in `pluginDefaults` object, so should hopefully be clear that it's forcing the default format for base64?)

Reply:
That's what I was missing :) Thank you for the explanation! 👍 🌮